.github/workflows/ci.yml

```diff
@@ -1,45 +1,52 @@
 name: Continuous Integration
 
 on:
-  push:
-  pull_request:
+  pull_request_target:
+    branches: [ master ]
 
 jobs:
   htmlproofer:
     name: HTMLProofer
+
     runs-on: ubuntu-latest
-    
+
     steps:
       - name: Checkout EIP Repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-      
+        with:
+          ref: "refs/pull/${{ github.event.number }}/merge"
```

Comment:
Oh this is neat, so it'll run with the workflow file from `master`, but fetch from the PR?

Reply:
Yes